weblate/machinery/tests.py

```diff
@@ -546,8 +546,15 @@ def mock_response(self) -> None:
     @responses.activate
     def test_translations_cache(self) -> None:
         self.mock_response()
-        machine = self.MACHINE_CLS(self.CONFIGURATION)
-        machine.delete_cache()
+        machine = self.get_machine(cache=True)
+
+        machine.validate_settings()
+        self.assertEqual(len(responses.calls), 2)
+        self.assertIn("langpair", responses.calls[1].request.params)
+        self.assertEqual("eng|spa", responses.calls[1].request.params["langpair"])
+        responses.reset()
+
+        self.mock_response()
```

Comment:
This really should go into a separate test case and not mixed witch cache testing.